Gemfile.lock

```diff
@@ -979,7 +979,7 @@ DEPENDENCIES
   web-console (~> 3.5)
 
 RUBY VERSION
-   ruby 2.7.1p83
+   ruby 2.7.5p203
```

Comment:
Is it required to update the ruby version or is it just a misconfiguration from a local environment ? 

PS : It should be harmless but it will add some step in the delivery process on server.

Reply:
Thanks! I updated the ruby  version